Users: Expire old user invites (#27361)
grafana/grafana: docs/sources/administration/configuration.md

Editors can administrate dashboards, folders and teams they create.
Default is `false`.

### user_invite_max_lifetime_duration

> The duration in time a user invite can be accepted before expiring. 

Reviewer: ```suggestion
The duration in time a user invitation remains valid before expiring. 
```

Reviewer: If you accept this change, please update it elsewhere as well.
